### DISCUSSION TITLE
this-week-in-neovim/77

### DISCUSSION THREAD

**Discussion by giscus** [codicocodes/dotfyle#144] [This Week in Neovim]
# this-week-in-neovim/77

This Week in Neovim 77 with news and updates from the Neovim plugin ecosystem and Neovim core.

https://dotfyle.com/this-week-in-neovim/77

**Comment by technbuzz** [comment]
Thanks for this series of posts.
Most of this series provides plugins, I like the minimal config, with absolute necessary plugins. I would mostly like to enhance or be aware of the native capabilities.

Would be nice if resources around that are also included in future.
